homework-20-12-23/argishti-bazikyan/.env.example

```diff
@@ -0,0 +1,2 @@
+AWS_ACCESS_KEY_ID=your_access_key_id
+AWS_SECRET_ACCESS_KEY=your_secret_access_key
```

Comment:
this can be skipped if we run aws configure after the aws-cli installation. Access key, Secret key, region and default output type can be set there.